canvas_modules/harness/cypress/support/e2e.js

```diff
@@ -31,6 +31,7 @@ import "./canvas/utils-cmds";
 import "./canvas/verification-cmds";
```

Comment:
also add the entire cypress directory.  It should have flagged this well in the build.